shell/platform/darwin/macos/framework/Source/FlutterApplication.mm

```diff
@@ -5,10 +5,11 @@
 #import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterApplication.h"
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterApplication_Internal.h"
 
+#import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterAppDelegate.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterAppDelegate_Internal.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h"
+
```

Comment:
Delete the spare line here.

Reply:
Ahh, OK.  I thought I should separate the C++ from the Objective C headers.